### PR TITLE
Add -nonormalize option to disable normalize function

### PR DESCRIPTION
**Pull request by Bankers88**
https://github.com/GitTools/GitVersion/issues/1701#issuecomment-507222625

### CODE REVIEW

**Comment by Bankers88** [comment]
Does any know why the Linux build fails azure pipeline?

**Comment by arturcic** [comment]
Looks like they have updated the Linux image https://github.com/microsoft/azure-pipelines-image-generation/compare/ubuntu16/156.2...releases/ubuntu16/156#diff-f87405c9dd43f633a04dd078666d61afR106, they updated the mono and that's the reason

**Comment by Bankers88** [comment]
Ping @asbjornu

**Comment by arturcic** [comment]
I guess he's on vacation at the moment.

**Comment by Bankers88** [comment]
> Sorry for the erogenous approval. This is missing documentation, isn't it?

This option is explained in the HelpWriter.cs. Or do you mean in de docs folder? If so where do you want this documentation?

**Comment by asbjornu** [comment]
@Bankers88, yes in the `docs` folder. Sorry for not being more specific.

**Comment by Bankers88** [comment]
Yet again an update of the Linux image?

**Comment by arturcic** [comment]
It seems it's now related to the way some parallel jobs are scheduled, I guess they deployed a new version of the engine and that one is not working as before

**Comment by arturcic** [comment]
@Bankers88, the build was fixed now, this time the Agent Name of the Azure pipeline was changed, and we rely on the name to understand if we run on azure pipelines.

**Comment by Bankers88** [comment]
@asbjornu documentation is updated!

File: docs/build-server-support/build-server-support.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ GitVersion has support for quite a few build servers out of the box. Currently w
...
  - [Team Build (TFS)](build-server/teambuild.md)
  - [TFS Build vNext](build-server/tfs-build-vnext.md)
- 
+
 When GitVersion.exe is run with the `/output buildserver` flag instead of outputting Json it will export variables to the current build server.
 For instance if you are running in TeamCity after you run `GitVersion /output buildserver` you will have the `%system.GitVersion.SemVer%` available for you to use
 
 When running in MSBuild either from the [MSBuild Task](/usage/msbuild-task) or by using the `/proj myproject.sln` parameter, GitVersion will make the MSBuild variables available in the format `$(GitVersion_SemVer)`.
+
+Standard GitVersion.exe normalize the branches if there is a build server detected. This behavior can be disabled with the /nonormalize option.
```
Comment (asbjornu) [inline docs/build-server-support/build-server-support.md]:
```suggestion
Standard GitVersion.exe normalize the branches if there is a build server detected. This behavior can be disabled with the `/nonormalize` option.
```

**Comment by asbjornu** [comment]
Thank you so much for your contributions, @Bankers88! 🙏

**Comment by fedebn** [comment]
I'm sorry, but I havn't understand how to use the /nonormalize option in my jenkins build.
Someone can explain?

**Pull request merged**